vault/barrier_aes_gcm.go

```diff
@@ -74,9 +74,9 @@ type AESGCMBarrier struct {
 // the provided physical backend for storage.
 func NewAESGCMBarrier(physical physical.Backend) (*AESGCMBarrier, error) {
 	b := &AESGCMBarrier{
-		backend:                  physical,
```

Comment:
Please check what `gofmt` binary you're using.

Reply:
Or your editor config

Reply:
Editor was using `goimports` as a format command. Replaced it with `gofmt` now.